ci/docker_entrypoint.sh

```diff
@@ -21,7 +21,8 @@ cd /opt && \
 cd /meta-datadog-config
 git config user.name 'CZI Datadog Management'
 git config user.email '<>'
-git commit --all --message 'Backup of latest Datadog state'
+git add --all
```

Comment:
Ah, of course! :)

Reply:
Because `commit --all` just ain't what ya' think! Ha.